future.tex

```diff
@@ -0,0 +1,83 @@
+\subsection{Future Projections}
+
+\begin{frame}
+	\frametitle{Future Nuclear Projections}
```

Comment:
Projections are the future, it's clear we're talking about nuclear stuff... can this frame title have more information with fewer words?